crates/uv-resolver/src/resolution.rs

```diff
@@ -543,12 +544,15 @@ pub struct DisplayResolutionGraph<'a> {
     /// The style of annotation comments, used to indicate the dependencies that requested each
     /// package.
     annotation_style: AnnotationStyle,
+    /// External sources for each package: requirements, constraints, and overrides.
+    sources: BTreeMap<PackageName, BTreeSet<SourceAnnotation>>,
```

Comment:
I changed this to use `BTreeSet` inside so that we don't need to call `.sorted` on the string representations. Instead, the values will already be sorted based on the sort order of `SourceAnnotation`.